.github/workflows/test.yml

```diff
@@ -43,14 +45,19 @@ jobs:
         vcs-repo-file-url: https://raw.githubusercontent.com/ros2/ros2/master/ros2.repos
         colcon-defaults: |
           {
+            rosbag2_path=$(colcon list -p --packages-select rosbag2)/..
+            rosbag2_packages=$(colcon list -n --base-paths ${rosbag2_path})
             "build": {
               "cmake-args": [
-                "-DCMAKE_CXX_FLAGS=\"-Werror\""
+                "-DCMAKE_CXX_FLAGS=\"-Werror\"",
+                "-DBUILD_ROSBAG2_BENCHMARKS=1"
               ]
             },
             "test": {
               "ctest-args": ["-LE", "xfail"],
-              "pytest-args": ["-m", "not xfail"]
+              "pytest-args": ["-m", "not xfail"],
+              "packages-select": [${rosbag2_packages}],
+              "packages-skip": [rosbag2_performance_benchmarking]
```

Comment:
@MichaelOrlov this is not a script. You must write the package names manually under `"build": { ... }` and `"test": { ... }` like I said in https://github.com/ros2/rosbag2/pull/1268/files#r1148171130